### ISSUE TITLE
Addition of Beer Advocate API

### ISSUE THREAD

**Issue by eastjordan** [SpencerHanson/discord.tools#6]
I was wondering if their will be an addition of the beer advocate api for this discord bot that could be utilized.

**Comment by SpencerHanson** [comment]
Yes, I will work on intergrating with that API in the coming weeks
